File: clients/php/src/Exception/CapabilityDeniedError.php
<?php

declare(strict_types=1);

namespace Priostack\Acn\Exception;

/** {@code capability-denied} — missing capability, or an invalid/revoked token. */
class CapabilityDeniedError extends AcnToolError
{
    public const OUTCOME = 'capability-denied';
}

File: clients/java/src/main/java/com/priostack/acn/CapabilityDeniedError.java
package com.priostack.acn;

import com.fasterxml.jackson.databind.JsonNode;

/** {@code capability-denied} — missing capability, or an invalid/revoked token. */
public class CapabilityDeniedError extends ACNToolError {

    private static final long serialVersionUID = 1L;

    public CapabilityDeniedError(String outcome, String detail, String method, JsonNode data) {
        super(outcome, detail, method, data);
    }
}
